tests/unit/account/test_workspaces.py

```diff
@@ -131,3 +71,23 @@ def test_client_for_workspace():
     specified_workspace_client = wrksp.client_for(Workspace(cloud="azure", deployment_name="adb-123.10"))
     assert "azure-client-secret" == specified_workspace_client.config.auth_type
     assert "https://adb-123.10.azuredatabricks.net" == specified_workspace_client.config.host
+
+
+def test_workspace_clients(workspace_mock):
+    ws_clients = workspace_mock.workspace_clients()
+    assert len(ws_clients) == 2
+    assert ws_clients[0].config.auth_type == "azure-cli"
+    assert ws_clients[0].config.host == "https://abc.azuredatabricks.net"
+
+
+def test_configured_workspaces(workspace_mock):
+    ws_clients = []
+    for ws in workspace_mock.configured_workspaces():
+        ws_clients.append(workspace_mock.client_for(ws))
+
+    # test for number of workspaces returned
+    assert len(ws_clients) == 2
+
+    # test for cloud and deployment name
+    assert ws_clients[1].config.auth_type == "azure-cli"
```

Comment:
this auth type is failing tests, don't test for it

Reply:
Adding the mock back seems to allow the unit test to pass.